### DISCUSSION TITLE
Welcome to Definition Discussions!

### DISCUSSION THREAD

**Discussion by heathermeeker** [Open-Weights/Definition#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heathermeeker** [comment]
Interactions for this project are subject to the Mozilla Community Participation Guidelines. https://www.mozilla.org/en-US/about/governance/policies/participation/

**Comment by tribecabrasil** [comment]
I have recently discovered Open Source AI 1.0 and would like to express my gratitude for its existence.

[Link to [Open Source AI 1.0 definition](https://opensource.org/ai/open-source-ai-definition): https://opensource.org/ai/open-source-ai-definition]

The internet is replete with “freemium” content that misrepresents itself as Open Source. In reality, these offerings merely provide a “free student/individual/research license.”

A more transparent approach is required to address this issue.

[Link to Cloud Google’s use cases for Open Source AI, that's not an open source AI, it’s a freemium source: https://cloud.google.com/use-cases/open-source-ai]

**Reply by heathermeeker** [reply]
Thanks for commenting. Unfortunately I don't the OSI definition is making much headway, and probably can't make a real difference unless there is someone to actually certify models as open source. The OSI and their community are too fragmented to do this. 

Also, I think freemium is a different question, and freemium is fine as long as the terms are transparent. I think the explanation from Google in your link is pretty accurate, actually.
